### ISSUE TITLE
Works on my laptop not on my Mac Mini

### ISSUE THREAD

**Issue by atrauring** [coreybutler/fenix#58]
I installed Fenix on two Macs, on my laptop, the other a Mac Mini. I start up a server on both from exact same directory (an external disk that actually sits on Mac Mini and is mounted remotely on my Laptop).

Fenix works like a charm on my laptop.  On my mac mini I get a blank screen and when I click on link it says "Safari can't open page because server unexpectedly dropped the connection". All it says in log after I start server:  "Screenshot captured." If I knew where that screenshot was I'd drag it here.

The problem is same with firewall on or off, although since it's still internal figured firewall irrelevant.

Obviously I refer this on my Mac Mini so I can share externally.

What am I doing wrong?

PS great tool!

**Comment by coreybutler** [comment]
@atrauring - I'm glad you're finding Fenix useful enough to put on all those Macs!

As for the issue, are you running Fenix simultaneously on these different systems, i.e. on the laptop and the mac mini at the same time? 

The screenshot feature just generates the thumbnail of the site, so it's not for troubleshooting. Does Fenix still show the server as running when you get the error in Safari?

**Comment by atrauring** [comment]
Hi Corey

It says the server is running but there are no message in the log after the screenshot one.

I thought about the simultaneous thing so I quit Fenix on the laptop and just ran on mini.

I also tried deleting all the files created in my ~/Library on Mini and restarting just in case there was something corrupted.

I also tried running it off a directory on the local disk in case somehow the mount interfered although considering its a network on the laptop…

So I already tried to eliminate as many possible causes as possible.

Thanks


From: Corey Butler
Reply-To: coreybutler/fenix
Date: Sunday, March 1, 2015 at 20:09
To: coreybutler/fenix
Cc: Aron Trauring
Subject: Re: [fenix] Works on my laptop not on my Mac Mini (#58)


@atrauring<https://github.com/atrauring> - I'm glad you're finding Fenix useful enough to put on all those Macs!

As for the issue, are you running Fenix simultaneously on these different systems, i.e. on the laptop and the mac mini at the same time?

The screenshot feature just generates the thumbnail of the site, so it's not for troubleshooting. Does Fenix still show the server as running when you get the error in Safari?

**Comment by atrauring** [comment]
From: Aron Trauring
Date: Sunday, March 1, 2015 at 20:16
To: coreybutler/fenix, coreybutler/fenix
Subject: Re: [fenix] Works on my laptop not on my Mac Mini (#58)

Hi Corey

It says the server is running but there are no message in the log after the screenshot one.

I thought about the simultaneous thing so I quit Fenix on the laptop and just ran on mini.

I also tried deleting all the files created in my ~/Library on Mini and restarting just in case there was something corrupted.

I also tried running it off a directory on the local disk in case somehow the mount interfered although considering its a network on the laptop…

So I already tried to eliminate as many possible causes as possible.

Thanks


From: Corey Butler
Reply-To: coreybutler/fenix
Date: Sunday, March 1, 2015 at 20:09
To: coreybutler/fenix
Cc: Aron Trauring
Subject: Re: [fenix] Works on my laptop not on my Mac Mini (#58)


@atrauring<https://github.com/atrauring> - I'm glad you're finding Fenix useful enough to put on all those Macs!

As for the issue, are you running Fenix simultaneously on these different systems, i.e. on the laptop and the mac mini at the same time?

The screenshot feature just generates the thumbnail of the site, so it's not for troubleshooting. Does Fenix still show the server as running when you get the error in Safari?
